org_status/org_hosts/github.py

```diff
@@ -14,28 +14,30 @@ class GitHubOrg(OrgHost):
     HostName = 'github'
     StatusProvider = [TravisBuildStatus, AppVeyorStatus]
 
-    def __init__(self, token, group, **kargs):
+    def __init__(self, token, group, sync=True, **kargs):
         super().__init__(**kargs)
 
         self._group = group
-        self._token = GitHubToken(token)
-        self._org = GitHubOrganization(self._token, self._group)
-
         self._status_provider = []
         for i in enumerate(self.StatusProvider):
             self._status_provider.append(self.StatusProvider[i[0]](self._group))
 
+        self._sync = sync
+        if sync:
+            self._token = GitHubToken(token)
+            self._org = GitHubOrganization(self._token, self._group)
+
```

Comment:
Maybe something like

```suggestion
            else:
                self._nosync_repositories = kargs['repositories']
```